### PR TITLE
Medical lobby tweaks

### PR DESCRIPTION
**Pull request by MadmanMartian**
## About The Pull Request 

Reinforces the medical lobby doors, so it's not the preferred method of brute-entry. (You now have to get above 20 damage with a melee weapon to deal damage to them. Previously it was 8 damage.)

Adds some buttons to the medical front desk, so desk-operators can allow people access to the front areas with ease.

Changes some wiring around, so the reception APC cable doesn't lead from the lobby area needlessly.

Fixes a bug with plasma windows becoming invisible (Unatomic, but I was planning on adding plasma windows to the front desk until we realized that was just going to be negative gameplay.)

image

## Why It's Good For The Game

The medbay doors were so weak, they could be broken before a doctor comes from a screen away to actually open it.

## Changelog
:cl:
balance: The medbay doors have now been reinforced, after complaints from doctors of vagabonds breaking them down for free healthcare.
/:cl:

### CODE REVIEW

**Review by genessee596 — APPROVED** [review]
Delegate stamp of approval.

I already cleared this on the discord, so I'll do it here too. Encourage people to hack in or smash windows instead of breaking down the airlocks, which are ass to repair.

Or, you know, actually get someone to let them in.

Still needs code approval.

**Review by TorinoFermic — APPROVED** [review]
Seeing nothing broken with the map

**Comment by genessee596** [comment]
Delegate and maintainer approval. Merging now.

**Pull request merged**
